html/semantics/document-metadata/the-link-element/link-style-error-limited-quirks.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//" "">
+<title>link: error events in limited quirks mode</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src=/common/get-host-info.sub.js></script>
+<div id="log"></div>
+<script>
+[
+ ["same-origin", "resources/css.py"],
+ ["cross-origin", get_host_info().HTTP_REMOTE_ORIGIN + "/html/semantics/document-metadata/the-link-element/resources/css.py"]
+].forEach(originType => {
```

Comment:
Please deduplicate the code by moving it into a separate JS file.

Reply:
I guess that's reasonable. Do you think it's fine to make it depend on document.compatMode or do you think it should be something you call externally?

The other thing I was thinking of also testing is `@import`, but how that affects load/error events isn't really defined yet.

Reply:
Using `document.compatMode` should be fine.